packages/internals/src/index.ts

```diff
@@ -70,6 +70,7 @@ export { parseBinaryTargetsEnvValue, parseEnvValue } from './utils/parseEnvValue
 export { pick } from './utils/pick'
 export { platformRegex } from './utils/platformRegex'
 export { printConfigWarnings } from './utils/printConfigWarnings'
+export { serializeQueryEngineName } from './utils/serializeQueryEngineName'
```

Comment:
Probably better to move it there to keep things together

Reply:
Actually, I'd advise against this, as it would not allow us to observe the right output when explicitly snapshot testing on "binary" and "library" (we would just see `query-engine-NORMALIZED` everywhere, and at that point either we trust the output code 100%, or we might have typo regressions in the future in `getDmmf` and `getConfig`)